tools/omero/omero_roi_import.xml

```diff
@@ -0,0 +1,105 @@
+<tool id="omero_roi_import" name="OMERO ROI Import" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@"
+      profile="20.01" license="MIT">
+    <description> with ezomero </description>
+    <macros>
+        <token name="@TOOL_VERSION@">5.18.0</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <xrefs>
+        <xref type="bio.tools">omero</xref>
+    </xrefs>
+    <requirements>
+        <requirement type="package" version="3.0.1">ezomero</requirement>
+        <requirement type="package" version="2.2.2">pandas</requirement>
+        <requirement type="package" version="21.0.2">openjdk</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        python $__tool_directory__/omero_roi_upload.py
+        --input_file $input
+        --image_id $id
+        --user $__user__.extra_preferences.get('omero_account|username', $test_username)
+        --psw $__user__.extra_preferences.get('omero_account|password', $test_password)
+        --host $omero_host
+        --port $omero_port
+        --log_file $log
+    ]]></command>
+    <inputs>
+        <param argument="input" type="data" format="tabular" optional="false" label="Tab File with ROIs" help="Select ROIs Tabular file"/>
+        <param argument="id" type="integer" optional="false" label="Image ID where annotate the ROIs"/>
+        <param argument="omero_host" type="text" optional="false" label="OMERO host URL"/>
```

Comment:
An validator would be good to have here (and in the other tools), i.e. such that users can only enter valid host names. Could also be less strict, e.g. space free string consisting of `[\w-/:]` or something like this. Plus a validator checking that the string is not empty. 

Ideally implemented as xml macro.